roles/generate-docs/templates/README.md.j2

```diff
@@ -377,6 +378,40 @@ Below are the instructions for updating containers:
   * or update a single container: `docker-compose up -d {{ project_name }}`
 * You can also remove the old dangling images: `docker image prune`
 
+## Building locally
```

Comment:
Write this up as an external post and link to it. The docs are long enough already.